apps/wappush/js/si_sl_screen_helper.js

```diff
@@ -47,13 +44,13 @@ var SiSlScreenHelper = (function() {
   }
 
   function sssh_populateScreen(message) {
-    var _ = navigator.mozL10n.get;
-
     screen.hidden = false;
 
     // Populate the message
     if (message && !message.isExpired()) {
-      title.textContent = message.sender;
+      var l10nTitle = Utils.prepareMessageTitle(message);
```

Comment:
Yes, it's intended. This is something that we were missing and it made this header incoherent with the others.